src/bootstrap/src/utils/helpers.rs

```diff
@@ -552,3 +552,22 @@ where
 {
     input.as_ref().iter().map(|x| format!("{:02x}", x)).collect()
 }
+
+/// Create a `--check-cfg` argument invocation for a given name
+/// and it's values.
+pub fn check_cfg_arg(name: &str, values: Option<&[&str]>) -> String {
+    // Creating a string of the values by concatenating each value:
+    // ',values("tvos","watchos")' or '' (nothing) when there are no values.
+    let next = match values {
+        Some(values) => {
+            let mut tmp =
+                values.iter().map(|val| [",", "\"", val, "\""]).flatten().collect::<String>();
+
+            tmp.insert_str(1, "values(");
+            tmp.push_str(")");
+            tmp
+        }
+        None => "".to_string(),
+    };
```

Comment:
```suggestion
    let next = values
        .map(|values| {
            let mut tmp =
                values.iter().map(|val| [",", "\"", val, "\""]).flatten().collect::<String>();

            tmp.insert_str(1, "values(");
            tmp.push_str(")");
            tmp
        })
        .unwrap_or_default();
```

Reply:
I have a slight preference for being explicit here. It IMO conveys better in which case in produces a empty string.

Reply:
Main idea was to avoid doing match with 2 nested blocks. You can still do explicit empty string with unwrap_or or unwrap_or_else. I don't have any strong feelings btw, this is just a nit suggestion